src/pages/DisplayCards.tsx

```diff
@@ -1,23 +1,75 @@
 import React, { useState } from "react";
+import ReactTooltip from "react-tooltip";
```

Comment:
Is this a new library/component? Antd has a Tooltip component which you could use.

Reply:
Yeah, I added the tooltips in before we decided on using antd but changed it over now :)